test/test_test.go

```diff
@@ -81,6 +155,31 @@ func (d *draggable) DragEnd() {
 	d.wasDragged = true
 }
 
+type focusable struct {
+	widget.BaseWidget
+	focused bool
+}
+
+var _ fyne.Focusable = (*focusable)(nil)
+
+func (f *focusable) Focused() bool {
```

Comment:
nit: alphabetical ordering would put Focused below FocusLost

Reply:
Yeah, you’re right.